parsers/nmparser/read_shk_test.go

```diff
@@ -55,42 +53,62 @@ func TestReadParseShkLines(t *testing.T) {
 				"            9            1  7.45166E+00  4.47258E+01  7.93537E+01  6.04504E+01",
 				"            10           1  2.47776E+01  0.00000E+00  0.00000E+00  0.00000E+00",
 			},
-			expected1: "TABLE NO.     1: First Order Conditional Estimation with Interaction: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0",
-			expected2: "TABLE NO.     2: First Order Conditional Estimation with Interaction: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0",
-			expected3: "TABLE NO.     3: First Order Conditional Estimation with Interaction: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0",
+			expectedTables: []string{
+				"TABLE NO.     1: First Order Conditional Estimation with Interaction: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0",
+				"TABLE NO.     2: First Order Conditional Estimation with Interaction: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0",
+				"TABLE NO.     3: First Order Conditional Estimation with Interaction: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0",
+			},
 			context:   "3 methods",
 			etaCount:  4,
 			epsCount:  1,
 		},
+		{
+			lines: []string{
```

Comment:
same point as above. These tests don't actually get run except periodically by a developer manually. Better to include in babylontest.